src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -264,10 +264,10 @@ public void testUpdateFeedbackResponseComment() throws Exception {
     }
 
     @Test
-    public void testDeleteFeedbackResponseComment() throws Exception {
+    public void testDeleteFeedbackResponseCommentById() throws Exception {
         //create a frComment to delete
         FeedbackResponseCommentAttributes frComment = restoreFrCommentFromDataBundle("comment1FromT1C1ToR1Q1S1C1");
-        frComment.setId(null);
+        frComment.setId(1L);
         frComment.feedbackQuestionId = getQuestionIdInDataBundle("qn2InSession1InCourse1");
         frComment.feedbackResponseId = getResponseIdInDataBundle("response2ForQ2S1C1", "qn2InSession1InCourse1");
```

Comment:
I don't think you need L270 - L272

Reply:
@xpdavid We need to retain L270, else there will be NPE at L281. Other two lines can be removed.

Reply:
But why you need `frComment.getId()`? You are testing `deleteFeedbackResponseCommentById` under the scenario that `silent fail nothing to delete`. So ideally, you can just supply an non-exist ID let's say `1234567`.

A more sophisticated method would be to use `getFeedbackResponseComment` to make sure `1234567` doesn't exist before the test.

Reply:
@xpdavid got it now.
Will modify this accordingly.

Reply:
@xpdavid I have made the change.